handler.py

```diff
@@ -321,11 +321,11 @@ def main():
 
         try:
             if choice == '1':
-                item_id = int(input("Enter item ID: "))
-                name = input("Enter item name: ")
                 price = float(input("Enter item price: "))
-                quantity = int(input("Enter item quantity: "))
-                store.add_item_to_inventory(item_id, name, price, quantity)
+                itemQuantity = int(input("Enter the item quantity please: "))
+                if itemQuantity < 0:
+                    itemQuantity = 0
+                store.add_item_to_inventory(price, itemQuantity)
```

Comment:
Add comment to every line.
```suggestion
             # Check if the user choice is '1'
            if choice == '1':
                # Prompt the user to enter item ID
                item_id = int(input("Enter item ID: "))
                # Prompt the user to enter item name
                name = input("Enter item name: ")
                # Prompt the user to enter item price
                price = float(input("Enter item price: "))
                # Prompt the user to enter item quantity
                quantity = int(input("Enter item quantity: "))
                # Add the item to the store inventory
                store.add_item_to_inventory(item_id, name, price, quantity)
                # Ask for the quantity of the item again
                itemQuantity = int(input("Enter the item quantity please: "))
                # Ensure item quantity is not negative
                if itemQuantity < 0:
                    itemQuantity = 0
                # Add the item to the store inventory with price and the final quantity
                store.add_item_to_inventory(price, itemQuantity)
```